src/secret/ledger_nano.rs

```diff
@@ -232,19 +231,18 @@ impl SecretManageExt for LedgerSecretManager {
                         None => return Err(crate::Error::InvalidBIP32ChainData),
                     };
                     (
-                        true,
                         Some(&a.address),
                         iota_ledger::LedgerBIP32Index {
                             bip32_change: remainder_bip32_indices[3] | HARDENED,
                             bip32_index: remainder_bip32_indices[4] | HARDENED,
                         },
                     )
                 }
-                None => (false, None, iota_ledger::LedgerBIP32Index::default()),
+                None => (None, iota_ledger::LedgerBIP32Index::default()),
             };
 
             let mut remainder_index = 0u16;
-            if has_remainder {
+            if remainder_address.is_some() {
```

Comment:
With `if let Some(remainder_address) = remainder_addresse{` we don't need to unwrap later `remainder_address.unwrap()`

Reply:
Yes please do this change as CI is complaining about it

Reply:
https://github.com/iotaledger/iota.rs/pull/1023/commits/5d4a31b7b2cf99c369799460ced792c1745f62bd